src/Conference/ConferenceDetails.test.js

```diff
@@ -17,36 +19,45 @@ jest.mock('react-router-dom', () => {
 
 configure({ adapter: new Adapter() });
 
+beforeAll(() => {
```

Comment:
Any reason why you removed the comment? Another developer might not understand why we're suppressing
warning at this point now :)

Reply:
Sorry the comment is still there in ours! We probably just didn't save it before we pushed, we'll push again!